### DISCUSSION TITLE
will you add support for xp pen star 05 ?

### DISCUSSION THREAD

**Discussion by owedits** [OpenTabletDriver/OpenTabletDriver#1738] [General]
that kinda is the tablet im using...

**Comment by jamesbt365** [comment]
Come over to our discord, we can't support tablets we don't have any information on so we require active communication with the person.

**Reply by jamesbt365** [reply]
We have the Star 05 V3?
https://github.com/OpenTabletDriver/OpenTabletDriver/blob/configs/OpenTabletDriver.Configurations/Configurations/XP-Pen/Star%2005%20V3.json
